docs/packages/reducers.md

```diff
@@ -60,7 +60,7 @@ Returns Redux state by action namespace.
 **Example**
 
 ```js
-import { getStateByAction } from '@redux-tools/reducers';
+import { getStateByAction } from '@redux-tools/namespaces';
```

Comment:
please move the documentation to the corresponding packages too. (and remove it from here)